src/utils/dataHandler.js

```diff
@@ -37,8 +37,26 @@ function getClassifications() {
     return classifications
 }
 
+// The function to go to the upper page
+function goUpperPage(classification) {
+    window.location.pathname = 'roles/' + classification + 'Roles'
+}
+
+// The function for navigation through roles pages
+function shiftRolesPage(index, x){
+    
+    const newIndex = index + x
+    if (newIndex === 0 || newIndex === 3){
+        window.location = 'townRoles'
+    } if (newIndex === 1) {
```

Comment:
not a clean way to write if conditions

try to get used to the habit of writing each if condition in a separate line and I always add a empty line between lines to increase the readability

```
    if (newIndex === 0 || newIndex === 3){
        window.location = 'townRoles'
    } 
    
    if (newIndex === 1) {
            window.location = 'mafiaRoles'
    }
    
    if (newIndex === 2 || newIndex === -1) {
        window.location = 'neutralRoles'
    }
```

Reply:
sure 👍